### DISCUSSION TITLE
Set Max SOC & Display Min SOC

### DISCUSSION THREAD

**Discussion by stevie445** [nathanmarlor/foxess_em#300] [Ideas]
Would it be possible to add something that allows the adjustment of the max SOC level?  I am not using anywhere near the battery capacity I have at this time of year (ranging around a SOC of 50%-70% at dawn without force charging) and would prefer to lower the max SOC and send the excess to my EV much sooner, then adjust it back up later in the day (through automation hopefully).  I need 1.4kW of excess before the EV charger kicks in, so it would be great to get this during the best part of the day.

If I can adjust the Max SOC, I can lower the Solar required before switching to charging my car.  I know I can do it on the inverter, but if I can, I would prefer to do it via HA.

Also, is there a way of viewing the min SOC set point on HA?  

Thanks!

**Comment by C00K50N** [comment]
If it's possible, I too would be interested in this as I'm keen to achieve a similar thing.

For now, I have created a couple automations which essentially switch my Zappi to Eco an hour or so after dawn, as long as the forecast remaining for the day is greater than the amount the battery still needs before full.  It's not perfect, but does allow the car to charge at 1.4kW for a while before the battery is full, at which point the car will then charge with all available excess.

In addition to answering the above @nathanmarlor, I wonder if it's also possible to get a value for (forecast remaining today minus average use remaining today)?  That would be more accurate than my current calculation which just looks at remaining capacity, not taking use into account.

**Comment by stevie445** [comment]
....Or even a 'pause battery charging' for a time period(???)
